tests/conftest.py

```diff
@@ -33,6 +34,12 @@ def standalone_tsm_context(standalone_tsm_context_com_object):
     return nitsm.codemoduleapi.SemiconductorModuleContext(standalone_tsm_context_com_object)
 
 
+class PublishedDataType(enum.Enum):
+    Double = 0
```

Comment:
There are constants in the standalone context type library (_standalone_tsm_context_tlb) for these. Can you access those here by using something like what is described here? https://www.oreilly.com/library/view/python-programming-on/1565926218/ch12s03s05.html

It's not a big deal if it's too difficult to get those constants.

Reply:
We would have to use makepy to statically bind the enum.  We'd end up with a file similar to `_pinmapinterfaces.py` built explicitly for the standalone context.  We could do that, but it also feels weird moving away from native python types and instead returning a com type when the `type` property is accessed.

Reply:
Here's a glimpse of what makepy generates. It really doesn't do us much good since the user will have to compare the integer value with `constants.PublishedDataType_Boolean`
![image](https://user-images.githubusercontent.com/51342134/139492635-570b2fb0-e235-480d-82e1-14bca99cc362.png)

Reply:
I'm fine with the original code.